pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -16,18 +15,55 @@ The configuration file enables you to import content from a Git repository into
 
 The {ProductShortName} configuration file defines the following parameters:
 
-* `server` The URL of the {ProductShortName} server.
-* `repositories` A multi-valued structure with the following child members:
+`server`:: (Required) The URL of the {ProductShortName} server.
+`repository`:: (Required) The name of the repository. It is recommended not to include spaces in the repository name.
+`variants`:: (Required) Separate sets of output built on the same set of files and differentiated by these values: `name`, `path`, and `canonical`. Modules and assemblies can have variants to differentiate how they are used, and what output is created.
 +
-** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies.
+** `name` (Optional) A unique ID that is displayed in URLs on the {ProductShortName} site when you view modules and assemblies. If no value is provided for `name`, this field is defined as *DEFAULT*.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
-** `modules` (optional) Patterns for modules that show up in the search results on {ProductShortName}.
-** `resources` (optional) Image files, legal information, or other content that is not a complete module or assembly.
-** `attributes` (optional) A path to the repository-wide attributes file. Provide this value if your documents relies on the attributes file.
-+
-NOTE: You must define the values for `server` and `repository`.
+`\https://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
+** `path` (Optional) The path containing resource files, such as `attributes.adoc`.
```

Comment:
`path` is not an optional filed, it is required as per the recent changes

Reply:
Fixed.